src/helpers/assign-pr-reviewers.ts

```diff
@@ -36,12 +36,19 @@ export const assignPrReviewers = async ({
   pull_number = String(context.issue.number)
 }: AssignPrReviewer) => {
   const coreMemberLogins = await getCoreMemberLogins(context.issue.number, teams?.split('\n'));
+  const {
+    data: { user }
+  } = await octokit.pulls.get({ pull_number: context.issue.number, ...context.repo });
 
   if (login && coreMemberLogins.includes(login)) {
     core.info('Already a core member, no need to assign.');
     return;
   }
-  const assignees = sampleSize(coreMemberLogins, Number(number_of_assignees));
+  let filteredCoreMemberLogins = coreMemberLogins;
+  if (user) {
+    filteredCoreMemberLogins = coreMemberLogins.filter(userName => userName != user.login);
+  }
+  const assignees = sampleSize(filteredCoreMemberLogins, Number(number_of_assignees));
```

Comment:
```suggestion
  const prAuthorUsername = user?.login;
  const filteredCoreMemberLogins = coreMemberLogins.filter(userName => userName !== prAuthorUsername);
  const assignees = sampleSize(filteredCoreMemberLogins, Number(number_of_assignees));
```

Reply:
Even if `prAuthorUsername` is undefined (which I don't think will ever be the case 🤷 ), this filter should still work as expected since none of the `coreMemberLogins` will be undefined